nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/bootstrap.conf

```diff
@@ -63,6 +63,20 @@ nifi.bootstrap.sensitive.key=
 # HashiCorp Vault Sensitive Property Providers
 nifi.bootstrap.protection.hashicorp.vault.conf=./conf/bootstrap-hashicorp-vault.conf
 
+# Note: the following mapping properties only apply if a Sensitive Property Provider that uses property contexts
+# is configured.  Otherwise, these values are ignored.
+#
+# If no nifi.bootstrap.protection.xml.context.location.mapping.* properties are provided, the context for protected
+# properties uses their filename as a location prefix, e.g. "authorizers.xml||Manager Password".
+# This creates a separate context for each unique property name in each XML configuration file.
+#
+# However, to reuse the same context in a more logical fashion, context mappings may be provided, in the format:
+# nifi.bootstrap.protection.xml.context.location.mapping.<contextLocation>=<identifier matching regex>
+# With the following configuration, for example, any XML property named "Manager Password" located inside
+# an XML block whose <identifier> starts with "ldap-" will be mapped to the context named "ldap||Manager Password",
+# regardless of whether it resides in authorizers.xml or login-identity-providers.xml.
+nifi.bootstrap.protection.xml.context.location.mapping.ldap=ldap-.*
```

Comment:
This explanation is helpful. Perhaps instead of defaulting to include a context, what about defaulting to no context or an internal default context?  That would allow values to be shared, but also allow differentiating them if necessary. Is there too much overlap in property naming for that to work?  Alternatively, having the default set of context name to file mapping in `bootstrap.conf` could provide a standard configuration without having to hard-coded locations in a `PropertyLocation` enum.

Reply:
That's a good point, there probably isn't much property name duplication, and letting them provide a mapping would take care of any that needed disambiguation without hard-coding anything.  I'll mull this over.  Thanks!